### ISSUE TITLE
Understanding Batch Normalization

### ISSUE THREAD

**Issue by yutayamazaki** [yutayamazaki/paper_reading#14]
# どんなもの
## 論文
Understanding Batch Normalization
https://arxiv.org/pdf/1806.02375.pdf

## 著者・所属機関
Johan Bjorck
Cornell University

## 投稿日
16 Sep 2018

# Abstract
- BatchNormalizationの解釈性について．
- BNめっちゃ効くけどなんでかについては一般的な解釈はない．

# わかったこと
- CIFAR10と100layer ResNetで検証．
- わりと一般論だがBNによって，学習率を上げることができ，学習が速く収束し，また精度も高い．
- BNの有無にかかわらず，lrが小さい(lr=0.0001)と学習も遅く，精度も低いことがわかった．(Fig.1)
- つまり，BNによってlrを上げられることが良い結果をもたらす．

- 式(2)から，batch_sizeとlrがSGDのノイズをコントロールする．
- batch_sizeはでかいほうが，学習にノイズを与えない．
- つまり，学習率が高いこととバッチサイズが大きいことは同じ役割かも．

- BNの一番の恩恵は，初期値の乱数による影響を取り除けること．

# 議論はある？

# 次に読むべき論文は？
